projects/packages/blaze/src/js/editor.js

```diff
@@ -99,12 +105,15 @@ const BlazePostPublishPanel = () => {
 				onClick={ trackClick }
 				onKeyDown={ trackClick }
 			>
-				<Button variant="secondary" href={ blazeUrl } target="_top">
+				<Button variant="secondary" href={ blazeUrl( postId ) } target="_top">
```

Comment:
You don't need to pass the postId because the `blazeUrl ` function doesn't expect it.

Reply:
Good point. This was a leftover :) Removed in 3fa209950f4c428007c1e7b363d3936e8e499a2e